cvat-ui/src/actions/boundaries-actions.ts

```diff
@@ -43,6 +43,7 @@ export function resetAfterErrorAsync(): ThunkAction {
             const state = getState();
             const job = state.annotation.job.instance;
 
+            dispatch(createAction(AnnotationActionTypes.DESTROY_CANVAS));
```

Comment:
Why do not put the `destroy` method call here itself, why do we need an additional reducer?

Reply:
Done